install/snowflake/install.yml

```diff
@@ -0,0 +1,16 @@
+id: snowflake
+name: Snowflake
+title: Snowflake monitoring
+description: |
+  Quickly monitor your Snowflake application by tracking performance issues with a full end-to-end view of distributed traces.
+target:
+  type: integration
+  destination: host
+  os:
+    - linux
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com/
```

Comment:
Looks like we need this URL to be updated with a specific link 😄

Reply:
Hi @josephgregoryii,
Yes, but they are not yet ready with the installation docs. Once the docs are ready I will update them and move to review.

Thank you...